src/figuresClasses.ts

```diff
@@ -1,11 +1,85 @@
-export interface Figure {}
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: string;
+  color: Color;
+  getArea(): number;
+}
```

Comment:
The `shape` field in the Figure interface should be of type `Shape` instead of `string` to ensure that only valid shapes are assigned.